README.md

```diff
@@ -1,87 +1,55 @@
+# HPCC Systems Platform Community Version
+  platform/
+  -  ca/base/<version>/Dockerfile: HPCC Systems Platform Community version prequisites (Ubuntu 18.04)
+     Docker Hub:  hpccsystems/hpcc-base
+  -  ce/Dockerfile  HPCC Systems Platform Community version Docker build file with version as input argument.
+     Docker Hub:  hpccsystems/platform
+  -  ln/base/<version>/Dockerfile: HPCC Systems Platform Internal version prequisites (CentOS 7)
+  -  ln/Dockerfile HPCC Systems Platform Internal with Plugins Docker build file with version as input argument
 
-# Docker HPCC
-### Supported tags and respective Dockerfile links
-* 7.0.18-1 latest  [(7/Dockerfile)](https://github.com/hpcc-systems/docker-hpcc/tree/master/7/Dockerfile)
-* 7.0.16-1
-* 7.0.14-1
-* 7.0.8-1
-* 7.0.6-1
-* 7.0.4-1
-* 7.0.2-1
-* 7.0.0-1
-* 6.4.40-1  [(6/Dockerfile)](https://github.com/hpcc-systems/docker-hpcc/tree/master/6/Dockerfile)
-* 6.4.38-1
-* 6.4.36-1
-* 6.4.32-1
-* 6.4.30-1
-* 6.4.28-1
-* 6.4.26-1
-* 6.4.24-1
+  clienttools/
+  -  ce/Dockerfile: HPCC Systems Clienttools Community Docker version build file with version as input argument
+     Docker Hub: hpccsystems/clienttools
+  -  ln/Dockerfile: HPCC Systems Clienttools Internal version Docker build file with version as input argument
 
+  plugins/
+  -  ce/Dockerfile: HPCC Systems Plugins Community version Docker build file (based Platform Docker image) with version as input argument
 
-## What is HPCCSystems (HPCC)
-The HPCC Systems server platform is a free, open source, massively scalable platform for big data analytics. Download the HPCC Systems server platform now and take the reins of the same core technology that LexisNexis has used for over a decade to analyze massive data sets for its customers in industry, law enforcement, government, and science.
+  spark/
 
-For more information and related downloads for HPCC Systems products, please visit https://hpccsystems.com
-
-
-## How to use Docker HPCC image
-You can start the Docker HPCC image in interactive (-i -t) or daemon mode (-d). You must start the HPCC processes then go to ECLWatch to submit jobs, query, and explore your data with the HPCC Systems platform.
-
-To map a docker container port to a host port.
-
-  "-p &lt;host port&gt;:&lt;docker container port&gt;"
-
-The default ECLWatch port is 8010.
-
-### Ubuntu
-To start Docker in interactive mode and map ECLWatch port 8010 to host 8010:
-```sh
-sudo docker run -t -i -p 8010:8010 hpccsystems/hpcc /bin/bash
+## How to buid
+Create HPCC Systems Platform Docker image :
+For example:
+```console
+sudo docker build -t hpccsystems/platform:7.4.8-1 --build-arg version=7.4.8-1 .
 ```
 
-To start Docker in daemon mode and map ECLWatch port 8010 to host 8020:
-```sh
-sudo docker run -d -p 8020:8010  hpccsystems/hpcc
-```
+# Development and custom Build
+  dev/<version>
+    <bionic|disco|eoan|el7|gcc7|gcc8|gcc9>/
```

Comment:
&lt;bionic is OK, but it might be good to put some space here as &lt; bionic | disco | eoan | etc | etc   more readable